test/renderers/test_parquet_renderers.py

```diff
@@ -0,0 +1,108 @@
+import io
+import pytest
+from abc import ABC, abstractmethod
+from test import test_volatility
+
+HAS_PYARROW = False
+try:
+    import pyarrow as pa
+    import pyarrow.parquet as pq
+    import pyarrow.compute as pc
+    HAS_PYARROW = True
+except ImportError:
```

Comment:
```suggestion
except ImportError:
   # The user doesn't have pyarrow installed, but HAS_PYARROW will be false so just continue
```
Last change to get it over the line...  5:)

Reply:
Thank you 😁